src/index.ts

```
@@ -0,0 +1,415 @@
+function parseHeaders(headerStr?: string): Record<string, string> {
+  const headers: Record<string, string> = {};
+  headers['Accept'] = 'application/json';
+  if (headerStr) {
+    headerStr.split(",").forEach((header) => {
+      const [key, value] = header.split(":");
+      if (key && value) headers[key.trim()] = value.trim();
+    });
+  }
+  return headers;
+}
+const path = require('path');
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)